project.clj

```diff
@@ -72,6 +72,7 @@
                  it.unimi.dsi/fastutil]]
    [com.draines/postal "2.0.3"]                                       ; SMTP library
    [com.jcraft/jsch "0.1.55"]                                         ; SSH client for tunnels
+   [org.apache.sshd/sshd-core "2.3.0"]                                ; SSH test server
```

Comment:
This is a test dependency, I didn't find a proper way to put it cc @camsaul

Reply:
The `:dev` profile is probably the best place to put it

Reply:
Updated, PTAL @camsaul